docs/using_the_android_sdk_bom.md

```diff
@@ -0,0 +1,38 @@
+# Using the Android SDK BOM in your own projects
```

Comment:
Content is really good but I will just reformat that to 2 headers:

1) (Concept) Using the maven BOM or something like above.
Explain here what we using etc.

2) (Procedure) Add SDK SDK
It's cleaner and faster for reader to just jump in and copy paste steps.  

This is pretty standard way how docs is formatted.

Reply:
Done 👍